hack/bcm_add_lite_nodes.py

```
@@ -0,0 +1,193 @@
+    return ctx
+
+
+def bcm_call(url, ssl_ctx, service, call, args=None):
+    payload = json.dumps({
+        "service": service,
+        "call": call,
+        "minify": True,
+        "args": args or [],
+    }).encode()
+
+    req = Request(url + "/json", data=payload)
```

Ruff (0.15.14)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)